Kernel/Memory/MemoryManager.cpp

```diff
@@ -426,6 +380,255 @@ UNMAP_AFTER_INIT void MemoryManager::parse_memory_map()
     });
 }
 
+UNMAP_AFTER_INIT void MemoryManager::parse_memory_map_fdt(MemoryManager::GlobalData& global_data, PhysicalPtr fdt_addr)
+{
+    auto fdt_header = *reinterpret_cast<DeviceTree::FlattenedDeviceTreeHeader const*>(quickmap_page(PhysicalAddress { fdt_addr }));
+    unquickmap_page();
+    // FIXME: Actually map this
+    u8* fdt_buffer_pointer = (u8*)__builtin_alloca(MUST(page_round_up(fdt_header.totalsize)));
```

Comment:
Especially since as I pointed out elsewhere before, real device FDTs are regularly above 70KiB:
![FDTs for ARM Chromebooks in the Linux Kernel](https://media.discordapp.net/attachments/1139481927594803260/1200100980214214809/image.png?ex=65c4f422&is=65b27f22&hm=2ddf971c7a24ac75c7713a4620b6ec66acbec1b68fefd18511328db2ca22bb61&=&format=webp&quality=lossless&width=1320&height=1365)

A "very large" fixed max size seems okay, but in that case we should provide a highly descriptive kernel panic message in case it's not enough to aid debugging.